filewatcher/filewatcher_test.go

```diff
@@ -95,8 +95,8 @@ func TestFileWatcher(t *testing.T) {
 func TestFileWatcherTimeout(t *testing.T) {
 	interval := time.Millisecond
 	filewatcher.InitialReadInterval = interval
-	round := interval * 10
-	timeout := round * 8
+	round := interval * 20
```

Comment:
What is this change about?

Reply:
on my local machine it takes right above 85ms for this one (it would pass if it's <85ms without any changes)